lib/web_ui/lib/src/engine/scene_view.dart

```diff
@@ -16,6 +16,20 @@ abstract class PictureRenderer {
   FutureOr<DomImageBitmap> renderPicture(ScenePicture picture);
 }
 
+class SceneRender {
```

Comment:
drive by comments

Does this class need to be public? Do the fields need to be public?

Reply:
Yeah everything can be private except the scene itself. I'll update it.